### PR TITLE
(SERVER-1539) Add dev driver to razor configuration

### PR DESCRIPTION
**Pull request by cprice404**

### CODE REVIEW

File: jenkins-integration/razor/README.md [hunk 1]
```diff
@@ -55,17 +58,17 @@ The files corresponding to the "broker types" are managed by Puppet; see the ops
...
-### Broker for the `driver` node
+### Broker for the `driver` nodes
 
-The `driver` node uses a broker called `pe`.  This broker was set up by ops and
+The `driver` nodes uses a broker called `pe`.  This broker was set up by ops and
 will handle the work of automatically bootstrapping a Puppet agent that is
 configured to talk to their PE master.
 
-Note that whenever this box is reprovisioned, it will end up creating a new
+Note that whenever these box are reprovisioned, they will end up creating a new
```
Comment (camlow325) [inline jenkins-integration/razor/README.md]: s/box/boxes

**Comment by camlow325** [comment]
🙈 Just the simple typo.

**Comment by cprice404** [comment]
typo fixed

**Comment by camlow325** [comment]
🚢

**Pull request merged**